.gitignore

```diff
@@ -2,5 +2,7 @@
 /dist/
 /toucan.css
 /index.css
+/theme-data.js
+/theme-data.d.ts
```

Comment:
Should we just build into `dist` so we don't need these additional ignores?

Reply:
Maybe. It'd be a breaking change for CDN consumers as their path would change.

Not uncommon to have dist in the CDN url tho